Decred Wallet/Extensions/UIColor.swift

```diff
@@ -26,6 +28,14 @@ extension UIColor {
         static let thinGray = UIColor.init(hex: "#a4abb1")
         static let darkGray = UIColor.init(hex: "#4e5f70")
         static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
+        static let orangeColor = UIColor.init(hex: "fd714a")
+        static let navigationBarColor = UIColor.init(hex: "689F38")
+        static let lightGrey = UIColor(red: 236.0/255.0, green: 238.0/255.0, blue: 241.0/255.0, alpha: 1.0)
+        static let lightBlue =  UIColor(red: 206.0/255.0, green: 238.0/255.0, blue: 250.0/255.0, alpha: 1.0)
+        static let separatorGrey = UIColor(red: 224/255, green: 224/255, blue: 224/255, alpha: 1.0)
+        static let greenishGrey = UIColor.init(hex: "F1F8E9")
+        static let textAmount = UIColor.init(hex: "#091440")
+        static let displayAmount = UIColor.init(hex: "617386")
```

Comment:
From the usages in the files affected by this move, I see that only the `textAmount` property is used. The rest can be safely deleted.
We'll also need to rename that property. What color is that really? Is it `grey`? `greyishBrown`? Let's find a better name for it.
Another thing to consider is if any of the other colors previously defined in the `appColors` struct can be used in place of this struct.

Reply:
resolved in f5d6a38